src/EventStream/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "EventStream Class Library",
+  "authors": [ "Stuart Blackler" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+
```

Comment:
At some stage we'll need to sort the tags above; I know I didn't do it with Components so I'll raise an issue to re-visit this later.